hooks/post_user_create

```diff
@@ -2,5 +2,4 @@
 
 user=$1
 
-sudo mkdir -p /home/$user
 sudo setfacl -m g:#GROUP#:rwx /home/$user
```

Comment:
Could you explain me what is `#GROUP#` and how it work ?

Reply:
This is something that already existed. The `#GROUP#` is replaced by the real application dedicated group during the installation [here](https://github.com/YunoHost-Apps/nextcloud_ynh/blob/master/scripts/install#L238).

Reply:
Ok, so why we don't use the something like `__GROUP__` witch is more standard ?

Anyway it's not the subject of this PR. So this PR is ok for me.